packages/terra-toolkit-docs/package.json

```
@@ -28,9 +28,12 @@
   },
   "homepage": "https://github.com/cerner/terra-toolkit",
   "scripts": {
-    "lint": "eslint --ext .js,.jsx ."
+    "compile": "babel src --out-dir lib --copy-files",
```

we can dot he following like our other monorepos: https://github.com/cerner/terra-core/blob/main/packages/terra-i18n/package.json#L36

This will remove the need to add another babel.config.js file in the terra-toolkit-docs package
```suggestion
    "compile": "babel  --root-mode upward  src --out-dir lib --copy-files",
```